show/platform.py

```diff
@@ -104,7 +104,8 @@ def psustatus(index, json, verbose):
 def ssdhealth(device, verbose, vendor):
     """Show SSD Health information"""
     if not device:
-        device = os.popen("lsblk -o NAME,TYPE -p | grep disk").readline().strip().split()[0]
+        _, stdout = getstatusoutput_noshell_pipe(["lsblk", "-o", "NAME,TYPE", "-p"], ["grep", "disk"])
+        device = stdout.strip().split()[0]
```

Comment:
>strip [](http://example.com/codeflow?start=24&length=5)

Do you change the behavior? Originally it only read first line, strip, and pick the first token.